www/src/css/_animations.scss

```diff
@@ -0,0 +1,45 @@
+@use './var' as *;
+
+$easeOutSine: cubic-bezier(0.61, 1, 0.88, 1);
+$easeInExpo: cubic-bezier(0.7, 0, 0.84, 0);
+$easeInOutSine: cubic-bezier(0.37, 0, 0.63, 1);
+$easeInOutCubic: cubic-bezier(0.65, 0, 0.35, 1);
+$easeInOutBack: cubic-bezier(0.68, -0.6, 0.32, 1.6);
+$easeInCirc: cubic-bezier(0.55, 0, 1, 0.45);
+$easeOutExpo: cubic-bezier(0.16, 1, 0.3, 1);
+$transition-fast: 200ms;
+$transition-medium: 500ms;
+$transition-slow: 1s;
+
+@mixin animation-hover-pop {
+  transition: transform 0.15s ease-out;
+  &:hover {
+    box-shadow: 0 2px 2px 0 #2e5e82aa;
+    transform: translateY(-1px);
+  }
+}
+
+@keyframes pulse {
+  0% {
+    color: #2a85ca;
+    border-color: #2a85ca;
+  }
+
+  100% {
+  }
+}
+
+@mixin animation-copy-button {
+  transition: transform 0.5s ease-in-out, width, background-color, transform,
```

Comment:
Super nerd opinion: I actually like using `linear` for light-based animations like color, opacity, etc. Because [computer color is logarithmic](https://www.youtube.com/watch?v=LKnqECcg6Gw) (`rgb(3, 3, 3)` is actually twice as bright as `rgb(2, 2, 2)` because of how human vision works), when you add a non-linear curve it can result in uneven transitioning of color values.

But for everything else—motion, etc.—non-linear values are better because they simulate physics, so things appear to move more naturally with momentum, acceleration, etc. But anyway, `linear` often gets a bad rap when it actually has a lot of uses

Reply:
(I know you didn’t ask I just wanted to talk about animation and no one asks 😭 )